### PR TITLE
[IMP] sale_order_line_price_history: turn into a widget

### PR DESCRIPTION
**Pull request by chienandalu**
This way, the user doesn't have to save the line first to see the
customer prices.

cc @Tecnativa TT29630

history-in-new-record

please review @Yajo @Tardo

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can we let it in black color as previously?

**Comment by chienandalu** [comment]
> Can we let it in black color as previously?

Oh, yes, sure. Done

**Review by MeetKD — APPROVED** [review]

File: sale_order_line_price_history/wizards/sale_order_line_price_history.py [hunk 1]
```diff
@@ -12,12 +12,19 @@ def _default_partner_id(self):
         line_id = self.env.context.get("active_id")
         return self.env["sale.order.line"].browse(line_id).order_partner_id.id
 
+    @api.model
+    def _default_product_id(self):
+        line_id = self.env.context.get("active_id")
+        return self.env["sale.order.line"].browse(line_id).product_id.id
```
Comment (pedrobaeza) [inline sale_order_line_price_history/wizards/sale_order_line_price_history.py:18]: Why do you need this if you are already passing `default_product_id` in the context?

**Review by pedrobaeza — APPROVED** [review]
Tested in runbot

File: sale_order_line_price_history/wizards/sale_order_line_price_history.py (hunk 1 above)
Comment (chienandalu) [reply sale_order_line_price_history/wizards/sale_order_line_price_history.py:18]: This was a related field. I preferred to keep the ability to call the wizard with just the line id in the context

**Pull request merged**